src/sas/sascalc/fit/pagestate.py

```diff
@@ -637,9 +637,11 @@ def _get_report_string(self):
                 chi2 = ("Chi2/Npts = " + value)
                 chi2_string = CENTRE % chi2
             elif name == "Title":
+                from sas.sasview.__init__ import __version__ as sasview_version
```

Comment:
Having sascalc depend on sasview is not ideal. 

sas.sasview depends on sas.sasgui which depends on sas.sascalc which depends on sas.sasview, so it is a bit circular.  

Not a technical problem in this case, since sas.sasview.__init__ doesn't import anything, but conceptually this in not clean.  

Should probably move version number to `sas.__init__` (but not for 4.2!). Add a ticket?

Reply:
BTW, line 31 already imports __version__ as SASVIEW_VERSION.